src/core/sensor/qgsiodevicesensor.h

```diff
@@ -261,6 +261,17 @@ class CORE_EXPORT QgsSerialPortSensor : public QgsIODeviceSensor
      */
     void setPortName( const QString &portName );
 
+    /**
+    * Returns the baudrate of the serial port the sensor connects to.
```

Comment:
When adding new functions to preexisting class, you need to add a \since QGIS X.XX documentation line.